Solutions/NetClean ProActive/Data Connectors/Connector_NetClean.json

```diff
@@ -0,0 +1,106 @@
+{
+    "id": "Netclean_ProActive_Incidents",
+    "title": "Netclean ProActive Incidents",
+    "publisher": "NetClean Technologies",
+    "descriptionMarkdown": "NetClean data connector provides the capability to ingest NetClean test data into Microsoft Sentinel.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Netclean_Incidents_CL",
+            "baseQuery": "Netclean_Incidents_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Netclean - All Activities.",
+            "query": "Netclean_Incidents_CL | sort by TimeGenerated desc"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Netclean_Incidents_CL",
+            "lastDataReceivedQuery": "Netclean_Incidents_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "Netclean_Incidents_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+        
+    },
+    "instructionSteps": [
+        {
+            "title": "",
+            "description": "1. How to get access to the data connector\n 2. If you have documentation to connect on your side link to that\n 3. Else, provide step by step instructions to discover the connection in your product\n",
```

Comment:
Hi @NCsteven please update the instruction steps in accordance with you Data connector, thanks!